Formula/rust.rb

```diff
@@ -79,6 +85,21 @@ def install
       system "make", "install"
     end
 
+    if build.with? "racer"
+      resource("racer").stage do
+        ENV.append_path "PATH", bin
```

Comment:
Does it fail without this? It looks like the calls below are fully qualified paths.

Assuming it is needed, probably want prepend not append.